inception-recommendation/src/main/java/de/tudarmstadt/ukp/inception/recommendation/tasks/TrainingTask.java

```diff
@@ -204,7 +202,7 @@ public void run()
                     ctx.close();
                     recommendationService.putContext(user, recommender, ctx);
                 }
-                catch (Throwable e) {
+                catch (RecommendationException | ConcurrentException e) {
```

Comment:
Same comment as in `SelectionTask`

Reply:
Catching Throwable is intentional here as we want to continue the execution even if a particular recommender fails.